.github/workflows/build.yml

```diff
@@ -36,7 +38,7 @@ jobs:
       - name: Build
         run: |
           mkdir -p build
-          go build -o build ./...
+          go build -ldflags="-X 'main.version=$(git describe --tags --abbrev=0)'" -o build ./...
```

Comment:
We probably don't need to do that here but in the Dockerfile that is actually being built for a specific release.

Reply:
That's right. But the dockerfile is not ready yet :)

Reply:
I did this for future testing, but it's not necessary either.